File: starting_with_c/count.py


def add_two(a: int, b: int):
    return a + b


def main(a: int):
    for i in range(a):
        add_two(1, 1)
        if (i == a - 1):
            print(i)


if __name__ == "__main__":
    num = input("")
    main(int(num))

File: starting_with_c/count.lua
local function add_two(a, b)
    return a + b
end


local i = io.read("n")
print(i)
for x = i,  0, -1 do
    add_two(1, 1)
    if x == 1 or x == (i/2) then
        print(x)
    end
end
